config/locales/administrate.pl.yml

```diff
@@ -6,6 +6,8 @@ pl:
       destroy: Usuń
       edit: Edytuj
       show: Wyświetl
+      new: Nowy
+      back: Plecy
```

Comment:
Almost! `plecy` is [part of body](https://www.fitomento.com/images/artykuly_fotol/Zacznij_dbac_o_swoje_plecy_trening_cwiczenia_na_plecy.jpg)